nextflow.config

```
@@ -52,6 +52,10 @@ params {
   cluster_weighting = "jaccard" // default weighting scheme for graph-based clustering
   nearest_neighbors = 20 // default nearest neighbors parameter for graph-based clustering
 
+  // Cell type annotation options
+  skip_celltyping = false // by default, we want to perform celltyping
```

```suggestion
  skip_celltyping = true // by default, we don't want to perform celltyping
```
I actually don't think we want to set this default to true. We want to skip cell typing unless we say to do it.